packages/modals/src/styled/StyledDrawerModalFooter.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'modals.drawer_modal.footer';
+
+export const StyledDrawerModalFooter = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: flex-end;
+  margin-top: auto;
```

Comment:
What is `margin-top` providing? Doesn't see like it is needed.

Reply:
I think at one point during spike it was used to keep the footer stickied to the bottom. However, it looks like it isn't needed and should be removed.